modules/account-lib/src/coin/trx/wrappedBuilder.ts

```diff
@@ -0,0 +1,92 @@
+import { Transaction } from './transaction';
```

Comment:
I'm not sure if we'll need this builder. we should circle up on the requirement

Reply:
We need it to maintain compatibilities with the current uses of account-lib in other projects like wallet-platform for example. 
Once we fully migrate to the factory model and update the projects that are using the "old builder" this class will be removed (we need to create an EPIC in jira to address that change)